### PR TITLE
Fix issues more (Angular)

### PR DESCRIPTION
**Pull request by groshenkovamarina**

### CODE REVIEW

File: packages/angular/src/app/pages/planning-task-list/planning-task-list.component.ts [hunk 1]
```diff
@@ -2,8 +2,10 @@ import {
   Component, OnInit, NgModule, Output, ViewChild, OnDestroy,
 } from '@angular/core';
 import { CommonModule } from '@angular/common';
+import { Router } from '@angular/router';
 import { DxButtonModule } from 'devextreme-angular/ui/button';
 import { DxDataGridModule } from 'devextreme-angular/ui/data-grid';
+// import { ExportingEvent } from 'devextreme/ui/data_grid';
```
Comment (dxvladislavvolkov) [inline packages/angular/src/app/pages/planning-task-list/planning-task-list.component.ts:8]: Remove comment

File: packages/angular/src/app/pages/planning-task-list/planning-task-list.component.ts [hunk 2]
```diff
@@ -24,11 +26,16 @@ import { TaskListGanttModule } from './task-list-gantt/task-list-gantt.component
 export class PlanningTaskListComponent implements OnInit, OnDestroy {
   @ViewChild('planningDataGrid', { static: false }) dataGrid: TaskListGridComponent;
 
+  @ViewChild('planningGantt', { static: false }) gantt: TaskListGanttComponent;
+
+  @ViewChild('planningKanban', { static: false }) kanban: TaskListKanbanComponent;
+
   @Output() tabValueChange = (e: TabsItemClickEvent) => {
```
Comment (dxvladislavvolkov) [inline packages/angular/src/app/pages/planning-task-list/planning-task-list.component.ts:33]: Remove output

Comment (dxvladislavvolkov) [reply packages/angular/src/app/pages/planning-task-list/planning-task-list.component.ts:33]: Do we need use event& Try to use active tab instead of event

**Review by dxvladislavvolkov — COMMENTED** [review]
Have some questions

File: packages/angular/src/app/pages/analytics-sales-report/analytics-sales-report.component.scss [hunk 3]
```diff
@@ -56,21 +52,7 @@
...
-    flex: 1 100%;
-  }
-}
-
-@media only screen and (max-width: 400px) {
-  .cards.wide > * {
-    flex: 1 320px;
-  }
-
+@media only screen and (max-width: 600px) {
```
Comment (GoodDayForSurf) [inline packages/angular/src/app/pages/analytics-sales-report/analytics-sales-report.component.scss:55]: may be better to use our classes for screen-sizes:
screen-large, screen-medium...

**Review by GoodDayForSurf — APPROVED** [review]

File: packages/angular/src/app/pages/analytics-geography/analytics-geography.component.ts [hunk 4]
```diff
@@ -1,5 +1,5 @@
 import {
-  Component, OnInit, NgModule, OnDestroy,
+  Component, OnInit, NgModule, OnDestroy, ViewChild,
```
Comment (dxvladislavvolkov) [inline packages/angular/src/app/pages/analytics-geography/analytics-geography.component.ts:2]: Remove ViewChild

File: packages/angular/src/app/pages/planning-task-list/task-list-kanban/task-list-kanban.component.ts [hunk 5]
```diff
@@ -23,6 +23,8 @@ type Board = {
   styleUrls: ['./task-list-kanban.component.scss'],
 })
 export class TaskListKanbanComponent implements OnChanges {
+  @ViewChild(DxSortableComponent, { static: false }) component: DxSortableComponent;
```
Comment (dxvladislavvolkov) [inline packages/angular/src/app/pages/planning-task-list/task-list-kanban/task-list-kanban.component.ts:26]: Rename component

**Review by dxvladislavvolkov — APPROVED** [review]

**Review by GoodDayForSurf — APPROVED** [review]

**Review by Shuvaevmedia — APPROVED** [review]

**Pull request merged**